internal/evaluator/conftest_evaluator.go

```diff
@@ -622,3 +657,46 @@ func ExtractStringFromMetadata(result output.Result, key string) string {
 	}
 	return ""
 }
+
+func withCapabilities(ctx context.Context, capabilities string) context.Context {
+	return context.WithValue(ctx, capabilitiesKey, capabilities)
+}
+
+// capabilities returns a JSON serialized OPA Capability meant to isolate rego
+// policies from accessing external information, such as hosts or environment
+// variables. If the context already contains the capability, then that is
```

Comment:
I was going to suggest some commentary to indicate "we do this for security reasons", but it's probably clear enough now.

Reply:
No, that's a good idea.

I renamed `capabilities` to `strictCapabilities` as per Joe's suggestion then added the following to the comment:
> The strict capabilities aim to provide a safe environment to execute arbitrary rego policies.